tests/integration/templates/internet_gateway.yml

```diff
@@ -0,0 +1,21 @@
+Resources:
+  Gateway:
+    Type: AWS::EC2::InternetGateway
+  Vpc:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.0.0.0/16
+  GatewayAttachment:
+    Type: AWS::EC2::VPCGatewayAttachment
+    Properties:
+      VpcId:
+        Ref: Vpc
+      InternetGatewayId:
+        !GetAtt [Gateway,InternetGatewayId]
+Outputs:
+  RefAttachment:
+    Value:
+      Ref: Gateway
```

Comment:
Something interesting to note: AWS claims this returns the "Name" which makes no sense for an `AWS::EC2::InternetGateway` :sweat_smile: